data/synthetic-blocks/synthetic-dataset-blocks.0.json

```diff
@@ -0,0 +1 @@
+{"metadata":{"type":"LBDatafile","rank":0},"phases":[{"id":0,"tasks":[{"entity":{"home":0,"id":1,"migratable":true,"type":"object"},"node":0,"resource":"cpu","time":0.5,"user_defined":{"shared_id":0,"shared_bytes":9.0,"home_rank":0}},{"entity":{"home":0,"id":3,"migratable":true,"type":"object"},"node":0,"resource":"cpu","time":0.5,"user_defined":{"shared_id":1,"shared_bytes":9.0,"home_rank":0}},{"entity":{"home":0,"id":2,"migratable":true,"type":"object"},"node":0,"resource":"cpu","time":0.5,"user_defined":{"shared_id":1,"shared_bytes":9.0,"home_rank":0}},{"entity":{"home":0,"id":0,"migratable":true,"type":"object"},"node":0,"resource":"cpu","time":1.0,"user_defined":{"shared_id":0,"shared_bytes":9.0,"home_rank":0}}],"communications":[{"type":"SendRecv","to":{"type":"object","id":5},"messages":1,"from":{"type":"object","id":0},"bytes":2.0},{"type":"SendRecv","to":{"type":"object","id":4},"messages":1,"from":{"type":"object","id":1},"bytes":1.0},{"type":"SendRecv","to":{"type":"object","id":2},"messages":1,"from":{"type":"object","id":3},"bytes":1.0},{"type":"SendRecv","to":{"type":"object","id":8},"messages":1,"from":{"type":"object","id":3},"bytes":0.5}]}]}
```

Comment:
I mentioned this in [vttv#80](https://github.com/DARMA-tasking/vt-tv/issues/80), but if we want to be able to use `shared_id`, `shared_bytes`, or `home_rank` as object QOI they should be in an `attributes` field instead of `user_defined`

Reply:
Indeed, but this brings the broader question as to whether vt will continue to output the `user_defined` field or not; in the former case then vt-tv should support this key as well.

Reply:
Ok, I've added support for `user_defined` in vttv ([PR 94](https://github.com/DARMA-tasking/vt-tv/pull/94))

Reply:
Perfect, thank you. Resolving conversation subsequently.